_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -0,0 +1,301 @@
+---
+id: b8bb68
+name: Text spacing in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+  This rule checks that text spacing set in the `style` attribute is below permissible minimum value or not set to `!important`
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any HTML element that is [visible][] and has one of the following CSS properties set by the `style` attribute:
+
+- [word-spacing][]
+- [letter-spacing][]
+- [line-height][]
+
+## Expectation 1
+
+The test target does not have "important" as its [word-spacing][] [priority][], unless the [computed][] [word-spacing][] is greater than `0.16` times the [computed][] [font-size][] of the test target.
+
+## Expectation 2
+
+The test target does not have "important" as its [letter-spacing][] [priority][], unless the [computed][] [letter-spacing][] is greater than `0.12` times the [computed][] [font-size][] of the test target.
+
+## Expectation 3
+
+The test target does not have "important" as its [line-height][] [priority][], unless the [computed][] [line-height][] is greater than `1.5` times the [computed][] [font-size][] of the test target.
```

Comment:
Rather than rely on the `getPropertyPriority()` API, I'd instead refer to the abstract [important flag](https://www.w3.org/TR/cssom/#css-declaration-important-flag). It's also important that we make sure to refer to the important flag of the _inline_ declaration, which is currently not the case. Otherwise, something like this would fail:

```html
<style>
  p { 
    line-height: 1.2 !important;
  }
</style>

<p style="line-height: 1.5">Hello world</p>
```

In fact, it's currently somewhat ambiguous what we mean by "`line-height` priority" as there can be multiple `line-height` declarations for the same test target, as seen above.